readme.md

```diff
@@ -98,3 +98,15 @@ Function to call when user selects an item. Item object is passed to that functi
 Type: `function`
 
 Function to call when user highlights an item. Item object is passed to that function as an argument.
+
+### displayDirection
+
+Type: `argument`
+
+Argument equal to `row` or `column`.
+
+### defaultColor / accentColor
```

Comment:
There are quite a few comments I have on this change to colors, could you extract these changes into a separate PR?